stories/FLocaleSelect.stories.ts

```diff
@@ -1,103 +1,104 @@
 import FLocaleSelect from '@/components/form/FLocaleSelect.vue';
-import FSelect from '@/components/form/FSelect.vue';
 import FButton from '@/components/FButton.vue';
-import { registerIcons } from '@/composables/useIcon';
-import { flagIcons } from '@/icons';
 
 import type { StoryFn } from '@storybook/vue3';
 import type { FLocaleSelectProps } from '@/components/form/FLocaleSelect.vue';
 
 export default {
   title: 'Components/Form/FLocaleSelect',
   component: FLocaleSelect,
+  argTypes: {
+    onClick: { action: 'click' },
+    onFocus: { action: 'focus' },
+    onBlur: { action: 'blur' },
+    onChange: { action: 'change' },
+    onInput: { action: 'input' },
+  },
 };
 
 const Template = (args: FLocaleSelectProps) => ({
-  components: { FLocaleSelect, FSelect },
-  setup: () => {
-    registerIcons('flags', flagIcons);
-    return { args };
-  },
-  template: `<FLocaleSelect v-bind="args" />`,
+  components: { FLocaleSelect },
+  setup: () => ({ args }),
+  template: '<FLocaleSelect v-bind="args" />',
 });
 
 export const Default: StoryFn<FLocaleSelectProps> = Template.bind({});
 Default.args = {
-  modelValue: 'FR',
+  modelValue: 'fr',
 };
 
 export const Small: StoryFn<FLocaleSelectProps> = Template.bind({});
 Small.args = {
   size: 'small',
-  modelValue: 'FR',
+  modelValue: 'fr',
 };
 
-export const LimitedLocales: StoryFn<FLocaleSelectProps> = Template.bind({});
-LimitedLocales.args = {
-  modelValue: 'FR',
-  locales: ['FR', 'AST', 'ES'],
+export const LimitedCountries: StoryFn<FLocaleSelectProps> = Template.bind({});
+LimitedCountries.args = {
+  modelValue: 'fr',
+  localeCodes: ['fr', 'es'],
 };
```

Comment:
You're in the FLocaleSelect story, countries should not be mentioned here I guess!